index.html

```diff
@@ -71,8 +70,210 @@ <h2>I'am a full-stack Web developer</h2>
                     </div>
                     <button><a href="https://popey17.github.io/planetrabbit/">See Project</a></button>
                 </div>
-             </div>
+            </div>
+            <div class="card-block">
+                <div class="card-pic">
+                    <div class="card-info">
+                        <div class="card-title">
+                            Aura Patient Record System
+                        </div>
+                        <div class="card-discription">
+                            A record system for patients info and treatment data at AURA health care clinic.
+                        </div>
+                        <div class="card-lang">
+                            <div>html</div>
+                            <div>css</div>
+                            <div>php</div>
+                            <div>laravel</div>
+                        </div>
```

Comment:
- [ ] Good job creating all the required elements to match with the Figma template 👍, but for [Semantics](https://www.w3schools.com/html/html5_semantic_elements.asp),  all lanuages (e.g., "bootstrap", "HTML" and "CSS") should be wrapped in `ul` and each of them a `li` 💯